templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Blazor/Pages/Index.razor

```diff
@@ -14,16 +14,41 @@
             <h5 class="m-1"> <i class="fas fa-rocket"></i> Congratulations, <strong>MyProjectName</strong> is successfully running!</h5>
         </Badge>
 
+        <Card>
```

Comment:
We should not commit such test code to GitHub. This is the startup template that is used when a developer wants to create a new solution, so we don't wan't to make it dirty :)

Reply:
I know this is just for testing. I was expecting for you to remove it after you review the UI :)